docs/src/SUMMARY.md

```diff
@@ -1,25 +1,34 @@
 # The Kani Rust Verifier
 
-- [Getting started with Kani](./getting-started.md)
+# Getting started
+
+  - [Introduction](./introduction.md)
   - [Installation](./install-guide.md)
-  - [Comparison with other tools](./tool-comparison.md)
-  - [Kani on a single file](./kani-single-file.md)
-  - [Kani on a package](./cargo-kani.md)
-  - [Kani result types](./kani-result-types.md)
-  - [Debugging failures]()
-  - [Debugging non-termination]()
-  - [Debugging coverage]()
+  - [Usage](./usage.md)
+    - [On a single file](./kani-single-file.md)
+    - [On a package](./cargo-kani.md)
+  - [Verification results](./verification-results.md)
 
-- [Kani tutorial](./kani-tutorial.md)
-  - [First steps with Kani](./tutorial-first-steps.md)
-  - [Failures that Kani can spot](./tutorial-kinds-of-failure.md)
-  - [Loops, unwinding, and bounds](./tutorial-loops-unwinding.md)
-  - [Creating non-deterministic variables](./tutorial-nondeterministic-variables.md)
+# Tutorial
+  - [Introduction](./kani-tutorial.md)
+  - [First steps](./tutorial-first-steps.md)
+  - [Failure detection](./tutorial-kinds-of-failure.md)
+  - [Loop unwinding](./tutorial-loop-unwinding.md)
+  - [Nondeterministic variables](./tutorial-nondeterministic-variables.md)
   - [Where to start on real code](./tutorial-real-code.md)
 
-- [Kani developer documentation](./dev-documentation.md)
-  - [Working with `RUSTC`](./rustc-hacks.md)
-  - [Testing](./kani-testing.md)
-  - [Book runner](./bookrunner.md)
+# Developer documentation
+  - [Command cheat sheets](./cheat-sheets.md)
+  - [Working with `rustc`](./rustc-hacks.md)
+  - [Testing](./testing.md)
+    - [Regression testing](./kani-testing.md)
+    - [Book runner](./bookrunner.md)
+
+# Limitations
+- [Guarantees]()
+- [Rust feature support](./rust-feature-support.md)
+- [Overrides](./overrides.md)
+---
+# Appendices
 
-- [Limitations](./limitations.md)
+- [Comparison with other tools](./tool-comparison.md)
```

Comment:
IMO this belongs with "getting started," the purpose is mostly to answer what Kani is and why people should be interested.

Reply:
I think of "getting started" as a section that you can read in a few minutes to get a quick understanding of the project. The comparison with other tools goes a bit deeper somehow and is linked in many sections around (including the introduction). In my opinion, this is an article that will be read by people who want to know more, but right after "Introduction" may cause potential users lose interest.